modules/core/test/test_io.cpp

```diff
@@ -584,6 +584,41 @@ struct data_t
     }
 };
 
+template <typename VecType>
+static void init_normal_2d_mat(Mat& mat_2d, int rows, int cols, int type, const Scalar& scalar)
+{
+    mat_2d = Mat(rows, cols, type, scalar);
+    for (int i = 0; i < rows; ++i)
+        for (int j = 0; j < cols; ++j)
+            mat_2d.at<VecType>(i, j)[1] = i + j;
+}
```

Comment:
it may be replaced with cv::randu output, right?

Reply:
Done